core/unsigneddata.go

```diff
@@ -180,6 +182,83 @@ func (b *VersionedBeaconBlock) UnmarshalJSON(input []byte) error {
 	return nil
 }
 
+type VersionedBlindedBeaconBlock struct {
+	eth2api.VersionedBlindedBeaconBlock
+}
+
+// NewVersionedBlindedBeaconBlock validates and returns a new wrapped VersionedBlindedBeaconBlock.
+func NewVersionedBlindedBeaconBlock(block *eth2api.VersionedBlindedBeaconBlock) (VersionedBlindedBeaconBlock, error) {
+	switch block.Version {
+	case spec.DataVersionBellatrix:
```

Comment:
Phase0 and Altair don't exist on the VersionedBlindedBeaconBlock struct so have been omitted from the switch statement

https://github.com/attestantio/go-eth2-client/blob/master/api/versionedblindedbeaconblock.go

Reply:
cool